state-chain/chains/src/lib.rs

```diff
@@ -344,6 +344,16 @@ pub enum AllBatchError {
 	Other,
 }
 
+#[derive(Debug)]
+pub enum ConsolidationError {
```

Comment:
Looks like we don't differentiate between these errors, anyway? are two variants required in that case?

Reply:
I would say it is still better to provide more information to the caller, and it servees as a form of documentation (it highlights the fact that there is a case other than "not required" which we might want to handle separately).